src/api/app/views/webui2/webui/user/register_user.html.haml

```diff
@@ -0,0 +1,12 @@
+:ruby
+  @pagetitle = 'Sign up'
```

Comment:
This title and the one on line 9 (`Sign Up for an Open Build Service account`) work well for a new user registering, but for an admin creating a user, it's a bit weird. Since the core of both pages (`Create User` under `Configuration` and `Sign Up` for new users) is the form, I would suggest having 2 views. They will be really similar, but also quite small and have a better title for each scenario.